semgrep/semgrep/core_exception.py

```diff
@@ -112,12 +114,13 @@ def into_semgrep_error(self) -> SemgrepError:
             )
             return SourceParseError(
                 short_msg="parse error",
-                long_msg=f"Could not parse {self._path.name} as {self._language}"
-                + (
-                    f"\n\nsemgrep-core message:\n\n\t{self._extra['message']}"
-                    if self._check_id != "ParseError" and "message" in self._extra
-                    else ""
-                ),
+                long_msg=f"Could not parse {self._path.name} as {self._language}",
```

Comment:
Is `self._extra['message']` ever informative with ParseErrors? Maybe worth leaving it in

Reply:
Note that before there was a `self._check_id != "ParseError"`. In the few tests I did I just got a "Syntax error" message which is not very useful.